optuna/artifacts/_download.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from optuna._experimental import experimental_func
+from optuna.artifacts._protocol import ArtifactStore
+
+
+@experimental_func("4.0.0")
+def download_artifact(
+    artifact_store: ArtifactStore,
+    artifact_id: str,
+    file_path: str,
+) -> None:
+    """Download an artifact from the artifact store.
+
+    Args:
+        artifact_store:
+            An artifact store.
+        artifact_id:
+            The identifier of the artifact to download.
+        file_path:
+            A path to download the artifact to.
+    """
```

Comment:
Could you add an entry for this function to the Artiface reference?
I'd like to check the rendered document.

https://github.com/optuna/optuna/blob/f954e3529d81ae156a9e91d31b50c89b390a3f67/docs/source/reference/artifacts.rst?plain=1#L20